### PR TITLE
Implemnt roundbox component

### PR DESCRIPTION
**Pull request by jchodev**
image

with 
image

### CODE REVIEW

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (AlbertoMApps) [inline core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt]: Note as discussed it can be reused after my merge button PR.

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (AlbertoMApps) [inline core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt]: Try to use params for these values that can change.

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (AlbertoMApps) [inline core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt]: The style can be stylable from the params too.

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (AlbertoMApps) [inline core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt:46]: Same with padding.

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (AlbertoMApps) [inline core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt:68]: I think paddings should be similar in each row.

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (AlbertoMApps) [inline core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt]: Extract this same as before, in case we want to change it later on.

**Review by AlbertoMApps — CHANGES_REQUESTED** [review]

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (jchodev) [reply core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt:46]: no, if same, it will be a big gap between two row

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (jchodev) [reply core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt]: I think just keep it is OK, due this component are ONLY using one area only

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (jchodev) [reply core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt:46]: I think in this moment is fine , because there have one area using this box only

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (jchodev) [reply core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt:68]: if all padding are same, the will have a big gap between two row

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (jchodev) [reply core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt]: Only one are using this compose, so we can do it later if need

File: core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt
Comment (AlbertoMApps) [reply core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundBoxWithTwoColumn.kt]: Are we talking about the colour here?

**Review by AlbertoMApps — APPROVED** [review]

**Pull request merged**
